fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -525,6 +539,7 @@ struct LogGenerationData : NonCopyable, public ReferenceCounted<LogGenerationDat
 					auto const& m = self->versionMessages.begin();
 					++messagesErased;
 
+					// TODO: understand what this means and make it work for txsTeam
```

Comment:
The `sizes` pair is (mutation data Bytes, txnStateStore mutation Bytes). if the team is `txsTeam`, then bytes is counted as the second part, otherwise the first part.

Reply:
thanks for explaining, yeah I wonder whether we can do the same or we need to change certain part to make new code path work -- kinda need a step back and look at who actually consume these values. TODO is to remind myself.